lib/licensed/dependency.rb

```diff
@@ -99,25 +89,16 @@ def remote_license_file
     # from the local LICENSE-type files, remote LICENSE, or the README, in that order
     def license_text
       content_files = Array(project.license_files)
-      content_files << remote_license_file if content_files.empty? && remote_license_file
+      content_files << remote_license_file if content_files.empty? && remote_license_file && remote_license_file.license.key == license_key
```

Comment:
if we're not using a remote file to find the license key, maybe we can 🔪 the remote file entirely?

Reply:
Fine with me. As I wrote in the commit message, it is a minor convenience:

> leave one convenience: pull license text from github if no local
> text and license on github is the same as asserted in package manager